app/controllers/api/databases.php

```diff
@@ -756,6 +760,9 @@ function updateAttribute(
             ->setParam('databaseId', $database->getId())
             ->setPayload($response->output($database, Response::MODEL_DATABASE));
 
+        $queueForUsage
+            ->addMetric(METRIC_DATABASES_STORAGE, 1); // Global, deletion forces full recalculation
```

Comment:
```suggestion
            ->addMetric(METRIC_DATABASES_STORAGE, CONST_1); // Global, deletion forces full recalculation
```
lets replace these with constants that we define in init.php